doc: Remove info about migrating active sessions, match the branding (#443)
TheThingsNetwork/docs: doc/content/the-things-stack/migrate-to-v3/_index.md

> ## V3 login

Reviewer: ```suggestion
## The Things Stack Community Edition login
```

> To be able to continue following steps below, you first need to log in to <a href="https://console.cloud.thethings.network/" target="_blank">The Things Stack Console</a>. You can do so by logging in with your The Things Network credentials via The Things ID.

Reviewer: ```suggestion
To be able to continue following steps below, you first need to log in to <a href="https://console.cloud.thethings.network/" target="_blank">The Things Stack Community Console</a>. You can do so by logging in with your The Things Network credentials via The Things ID.
```

> ## Add an application in V3

Reviewer: ```suggestion
## Add an application in The Things Stack
```

`ttn-lw-migrate` is used to export end devices and applications from The Things Network Stack V2 cluster to a <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/device-json/" target="_blank">JSON file</a>. This JSON file can later be <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/import-devices/" target="_blank">imported</a> in The Things Stack V3 via Console or via CLI.

### Note about migrating active sessions

> Starting from The Things Stack version `v3.12.0`, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a rejoin on the V3 network, and both OTAA and ABP devices will not have to be deleted from the V2 cluster in order to work properly when migrated to the V3 cluster. 

Reviewer 2: They will have to be deleted on V2

Author: I must have made a bad conclusion reading [this](https://www.thethingsindustries.com/docs/getting-started/migrating/migrating-from-v2/#export-end-devices-from-the-things-network-stack-v2):

> Migration from The Things Network Stack V2 to The Things Stack is one-way. LoRaWAN devices may only be handled by one Network Server at a time. The commands below will clear both the root keys and the session keys from The Things Network Stack V2 after the devices are exported. This means that the devices will no longer work in The Things Network Stack V2 .

I thought that migrating an active session means clearing those keys on V2, and that clearing those keys means you won't have to delete the device on V2 because it won't work anymore anyway.

`ttn-lw-migrate` is used to export end devices and applications from The Things Network Stack V2 cluster to a <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/device-json/" target="_blank">JSON file</a>. This JSON file can later be <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/import-devices/" target="_blank">imported</a> in The Things Stack V3 via Console or via CLI.

### Note about migrating active sessions

> Starting from The Things Stack version `v3.12.0`, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a new join on the V3 network. 

Reviewer: ```suggestion
Starting from The Things Stack version `v3.12.0`, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a new join on The Things Stack. 
```

> {{< warning >}} Migrating active sessions is **not available for The Things Network**, which means that your device will have to perform a new join. 

Reviewer: ```suggestion
{{< warning >}} Migrating active sessions is **not available for The Things Stack Community Edition**, which means that your device will have to perform a new join. 
```

> The reason for this is that after upgrading to The Things Stack V3, The Things Network started using a different `DevAddr` block than the one used on The Things Network clusters running The Things Network Stack V2.

Reviewer: ```suggestion
The reason for this is that a different `DevAddr` block is used in The Things Stack Community Edition compared to The Things Network V2.
```

Can we explain why we started using a new DevAddr block?

### Note about migrating an active session

Starting from The Things Stack version `v3.12`, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a rejoin on the V3 network, and both OTAA and ABP devices will not have to be deleted from the V2 cluster in order to work properly when migrated to the V3 cluster. 

> > **Important note:** Migrating active sessions is **not available for The Things Stack Community Edition**, i.e. you will not be able to migrate active sessions if you are using The Things Network. The reason for this is that The Things Stack Community Edition uses a different `DevAddr` block than the one used by The Things Network V2. 

Reviewer 2: Is it clear here that The Things Stack Community Edition is The Things Network?

This is needed because some recommended settings used by The Things Network V2 and The Things Stack Community Edition Network Server are different. For example, `RxDelay` parameter used by the The Things Network V2 is 1 second by default, while the recommended value for The Things Stack Community Edition is 5 seconds. The Things Stack Community Edition Network Server will try to automatically change this for you, but it might take some time until the MAC command reaches the end device. 

See the official The Things Stack documentation to learn how to <a href="https://www.thethingsindustries.com/docs/devices/mac-settings/" target="_blank">configure MAC settings</a>.

## Migrate your gateway

<a href="https://www.thethingsindustries.com/docs/getting-started/migrating/gateway-migration/" target="_blank">Migrating your gateway</a> from The Things Network V2 to The Things Stack Community Edition cluster is a really easy process. 

First, you need to <a href="https://www.thethingsindustries.com/docs/gateways/adding-gateways/" target="_blank">add a gateway to The Things Stack Community Edition</a>.

> Then, you need change the address of The Things Stack Community Edition cluster you are using - this will depend on which type of gateway you are using, <a href="https://www.thethingsindustries.com/docs/gateways/semtech-udp-packet-forwarder/" target="_blank">Semtech UDP Packet Forwarder</a> or <a href="https://www.thethingsindustries.com/docs/gateways/lora-basics-station/" target="_blank">LoRa Basics Station</a>. Let us assume you are using `eu1.cloud.thethings.network`.

Reviewer 3: ```suggestion
Then, you need to change the address of The Things Stack Community Edition cluster you are using - this will depend on which type of gateway you are using, <a href="https://www.thethingsindustries.com/docs/gateways/semtech-udp-packet-forwarder/" target="_blank">Semtech UDP Packet Forwarder</a> or <a href="https://www.thethingsindustries.com/docs/gateways/lora-basics-station/" target="_blank">LoRa Basics Station</a>. Let us assume you are using `eu1.cloud.thethings.network`.
```
